doc/website/docusaurus.config.js

```diff
@@ -147,9 +147,13 @@ module.exports = {
             from: '/dev-database',
           },
           {
-            to: '/concepts/ddl',
+            to: '/knowledge/ddl',
             from: '/ddl/intro',
```

Comment:
```suggestion
            from: ['/ddl/intro', '/concepts/ddl'],
```
instead of splitting it up into two - you can put the two previous docs in an array, and then remove lines 153-156